src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -472,8 +476,15 @@ ensogl::define_endpoints! {
         visualization_enable_fullscreen (NodeId),
         visualization_set_preprocessor  ((NodeId,data::EnsoCode)),
 
+        visualization_selected          (NodeId),
+        visualization_deselected        (NodeId),
```

Comment:
We have now better way of expressin it. Use this instead please:
```rust
on_visualization_select (Switch<NodeId>)
```
This will cover both `visualization_selected` and `visualization_deselected` and will simplify FRP network definition